client/x/epochs/module/module.proto

```diff
@@ -0,0 +1,12 @@
+syntax = "proto3";
+
+package client.x.epochs.module;
+
+import "cosmos/app/v1alpha1/module.proto";
+
+// Module is the config object of the epochs module.
+message Module {
+  option (cosmos.app.v1alpha1.module) = {
```

Comment:
why do you need this?

Reply:
This is basic config object for module. Other modules also have this proto file and auto-generated `module.pulsar.go`.